apps/web/src/components/forms/password.tsx

```diff
@@ -105,7 +133,7 @@ export const PasswordForm = ({ className }: PasswordFormProps) => {
           <Input
             id="current-password"
             type={showCurrentPassword ? 'text' : 'password'}
-            minLength={6}
+            minLength={8}
```

Comment:
You should leave it as `6` because this may prevent old passwords from being reset.

Reply:
Ahh! True, that would prevent current/existing users from logging in, thanks for pointing it out, will fix it in the next commit :)